fearless/Modules/Staking/SelectValidatorsFlow/SelectValidatorsStart/RecommendationsComposing.swift

```diff
@@ -1,8 +1,24 @@
+import Foundation
+
+protocol Recommendable {
+    var identity: AccountIdentity? { get }
+    var stakeReturn: Decimal { get }
+    var totalStake: Decimal { get }
+    var ownStake: Decimal { get }
+    var hasIdentity: Bool { get }
+    var oversubscribed: Bool { get }
+    var hasSlashes: Bool { get }
+    var blocked: Bool { get }
+}
+
 protocol RecommendationsComposing {
```

Comment:
What is the point of this complicated protocol. Firstly, I thought it should allow us to reuse some logic but still duplicating the clustering logic. Can we move clustering logic into the separate method of the protocol to reuse it in the implementations?